src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js

```diff
@@ -279,8 +283,12 @@ function paymentsDetails() {
           'Adyen-ShowConfirmation',
           'merchantReference',
           response.merchantReference,
+          'orderToken',
+          requestBody.orderToken,
```

Comment:
Do we need 2 orderTokens in this URL? If so, should we give them distinctive names?

Reply:
Had a merger malfunction, hence the two things you noticed. I'm fixing it.